features/append_snippets.feature

```diff
@@ -127,42 +25,56 @@ Feature: Append snippets option
           private $apples = 0;
           private $parameters;
 
-          public function __construct(array $parameters = array()) {
+          public function __construct(array $parameters = [])
+          {
               $this->parameters = $parameters;
           }
 
-          #[Given('/^I have (\d+) apples?$/')]
-          public function iHaveApples($count) {
+          #[Given('/^I have (\\d+) apples?$/')]
+          public function iHaveApples($count)
+          {
               $this->apples = intval($count);
           }
 
-          #[When('/^I ate (\d+) apples?$/')]
-          public function iAteApples($count) {
+          #[When('/^I ate (\\d+) apples?$/')]
+          public function iAteApples($count)
+          {
               $this->apples -= intval($count);
           }
 
-          #[When('/^I found (\d+) apples?$/')]
-          public function iFoundApples($count) {
+          #[When('/^I found (\\d+) apples?$/')]
+          public function iFoundApples($count)
+          {
               $this->apples += intval($count);
           }
 
-          #[Then('/^I should have (\d+) apples$/')]
-          public function iShouldHaveApples($count) {
-              \PHPUnit\Framework\Assert::assertEquals(intval($count), $this->apples);
+          #[Then('/^I should have (\\d+) apples$/')]
+          public function iShouldHaveApples($count)
+          {
+              PHPUnit\Framework\Assert::assertEquals(intval($count), $this->apples);
           }
 
           #[Then('/^context parameter "([^"]*)" should be equal to "([^"]*)"$/')]
-          public function contextParameterShouldBeEqualTo($key, $val) {
-              \PHPUnit\Framework\Assert::assertEquals($val, $this->parameters[$key]);
+          public function contextParameterShouldBeEqualTo($key, $val)
+          {
+              PHPUnit\Framework\Assert::assertEquals($val, $this->parameters[$key]);
+          }
+
+          #[Given('/^context parameter "([^"]*)" should be array with (\\d+) elements$/')]
+          public function contextParameterShouldBeArrayWithElements($key, $count)
+          {
+              PHPUnit\Framework\Assert::assertIsArray($this->parameters[$key]);
+              PHPUnit\Framework\Assert::assertEquals(2, count($this->parameters[$key]));
           }
 
-          #[Given('/^context parameter "([^"]*)" should be array with (\d+) elements$/')]
-          public function contextParameterShouldBeArrayWithElements($key, $count) {
-              \PHPUnit\Framework\Assert::assertIsArray($this->parameters[$key]);
-              \PHPUnit\Framework\Assert::assertEquals(2, count($this->parameters[$key]));
+          private function doSomethingUndefinedWith()
+          {
           }
 
-          private function doSomethingUndefinedWith() {}
+          private function useClasses(PyStringNode $node, TableNode $table)
```

Comment:
If we add some `use` statements we must have some code which actually uses these classes, otherwise the CS fixer will complain, so I added these dummy methods which just use these classes